### PR TITLE
Mb/05 11 19/case study related articles

### PR DESCRIPTION
**Pull request by meetbhatt23**

### CODE REVIEW

File: src/components/molecules/CardGroup.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components";
+
+import Card from "./Card";
+
+const CardGroupContainer = styled.ul`
+
```
Comment (tanvibhakta) [inline src/components/molecules/CardGroup.js]: Hello! This is an empty container, and the component has a class it isn't using. :) Could you clean up?

File: src/components/templates/case-studies/CaseStudyRelatedArticles.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        </CaseStudyRelatedArticlesContainer>
+    )
+};
+
+CaseStudyRelatedArticles.propTypes = {
+    heading: PropTypes.string.isRequired,
+    articles: PropTypes.array.isRequired,
+};
+
+CaseStudyRelatedArticles.defaultProps = {
```
Comment (tanvibhakta) [inline src/components/templates/case-studies/CaseStudyRelatedArticles.js]: Don't you need to set default props only in cases where the prop _isn't_ required?

Comment (meetbhatt23) [reply src/components/templates/case-studies/CaseStudyRelatedArticles.js]: Done.

File: src/components/molecules/CardGroup.js (hunk 1 above)
Comment (meetbhatt23) [reply src/components/molecules/CardGroup.js]: Removed this, thanks for noticing.

File: src/components/molecules/Card.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    max-height: calc(2* ${props => props.theme.lineHeights[5]}); // This needs to be in relation to line-height of the text component, currently BodyText3
+  }
+  .image {
+    margin-top: ${props => props.theme.spacings[3]};
+  }
+`;
+
+const CardContents = (props) => (
```
[the remark this answers is not in the document]
Comment (meetbhatt23) [reply src/components/molecules/Card.js]: Let's leave this be and merge the PR? We'll have a look at naming of all components sometime later this week.

**Comment by tanvibhakta** [comment]
Why does the shadow for the card not apply to the first element?

Screenshot 2019-11-11 at 4 24 50 PM

**Comment by tanvibhakta** [comment]
Also, the styles on this card component assumes that cards will only have content in this form. When I will be implementing the News card I will need to re-use this component, in which case it might be better to have a base card that this specific CaseStudies card draws from. Unless implementing bem naming
 will somehow solve this issue?

**Comment by meetbhatt23** [comment]
> Also, the styles on this card component assumes that cards will only have content in this form. When I will be implementing the News card I will need to re-use this component, in which case it might be better to have a base card that this specific CaseStudies card draws from. Unless implementing bem naming

No the card was made based on the assumption that it was the only sort of card we had. For the news section to use the card, you will need to refactor the card component. Also what does a card to anyway? This one doesn't even apply a border or a shadow. I'd say you make a new component called News card, since this one doesn't do anything anyway.

**Comment by meetbhatt23** [comment]
> Why does the shadow for the card not apply to the first element?
> 
> Screenshot 2019-11-11 at 4 24 50 PM

Those are the images from figma 🤷‍♂

**Pull request merged**